packages/cfd/src/features/Containers/maltainvest-real-account-signup.tsx

```diff
@@ -0,0 +1,242 @@
+import React from 'react';
+import { Div100vhContainer } from '@deriv/components';
+import { useIsAccountStatusPresent } from '@deriv/hooks';
+import { isDesktop, getAuthenticationStatusInfo } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import type { TCoreStores } from '@deriv/stores/types';
+import ProofOfAddress from '../Components/proof-of-address';
+import ProofOfIdentity from '../Components/proof-of-identity';
+import PersonalDetailsContainer from './personal-details-container';
+import { useCfdStore } from '../../Stores/Modules/CFD/Helpers/useCfdStores';
+import { JURISDICTION } from '../../Helpers/cfd-config';
+
+type TMaltainvestRealAccountSignupProps = {
```

Comment:
Name this component `real-account-signup-verifications`. These verifications are not just used for `maltainvest` accounts.